lib/event-listeners/foodsListeners.js

```diff
@@ -7,6 +7,29 @@ $(document).ready(function() {
   $('window').on('load', allFoods.getFoods())
 
   $('.foods-table').on('click', '.delete-food', deleteFood.deleteFood)
+  $('.new-food-form input[type="submit"]').on('click', createNewFood.createNewFood)
+
+  $('table td').keyup(function() {
+    clearTimeout($.data(this, 'timer'));
+    const wait = setTimeout(saveData(), 500); // delay after user types
+    $(this).data('timer', wait);
+  });
+
+  function saveData() {
+    const nameID = $(".foods-table input[name='update-id']").val();//change these inputs
+    const calorieID = $(".foods-table input[name='post-description']").val();
+
+    return $.ajax({
+      url: 'https://api-qs.herokuapp.com/api/v1/foods/:id',
+      method: 'PUT',
+      data: { food: {name: nameID, calories: calorieID} }
+    })//.done(function(data) {
```

Comment:
unless there's a reason to keep, I think the commented out lines should be deleted